drivers/usb-common.h

```diff
@@ -122,6 +122,10 @@
  #define USB_CTRL_MSGVALUE_MIN	0
  #define USB_CTRL_MSGVALUE_MAX	UINT16_MAX
 
+ typedef uint8_t usb_ctrl_cfgindex;
```

Comment:
May need a copy around line 420 for libusb-0.1 compatibility.

Reply:
Yeah, Appveyor noted this too:
````
n file included from nutdrv_qx.c:50:
nut_libusb.h:84:9: error: unknown type name 'usb_ctrl_cfgindex'
   84 |         usb_ctrl_cfgindex usb_config_index;             /* index of the device config we use. Almost always 0; see comments above */
      |         ^~~~~~~~~~~~~~~~~
cc1.exe: note: unrecognized command-line option '-Wno-unknown-warning-option' may have been intended to silence earlier diagnostics
cc1.exe: note: unrecognized command-line option '-Wno-reserved-identifier' may have been intended to silence earlier diagnostics
In file included from libusb0.c:34:
nut_libusb.h:84:9: error: unknown type name 'usb_ctrl_cfgindex'
   84 |         usb_ctrl_cfgindex usb_config_index;             /* index of the device config we use. Almost always 0; see comments above */
      |         ^~~~~~~~~~~~~~~~~
````

Reply:
Fixed (I hope) and pushed. I'll check in on this in the morning. 

I also need to get the rest of NUT setup and get this working with my DIY project!